src/frontend/templates/pages/sgfplayer.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% block title %}
+Game ID {{game.id}} (run {{game.run.name}})
+{% endblock %}
+
+{% block content %}
+
+{% if kind == "rating" %}
+<h1 class="title">Rating Game ID {{game.id}} (run {{game.run.name}}) </h1>
+{% else %}
+<h1 class="title">Training Game ID {{game.id}} (run {{game.run.name}}) </h1>
+{% endif %}
+
+<script type="text/javascript" src="/static/js/wgo.min.js"></script>
+<script type="text/javascript" src="/static/js/wgo.player.min.js"></script>
+<link type="text/css" href="/static/css/wgo.player.css" rel="stylesheet" />
+<link type="text/css" href="/static/css/wgo-custom.css" rel="stylesheet" />
```

Comment:
Ideally we put this a in a a `{% block X %}` (X being maybe "Head", not sure if the is one think that exists like this right now) so the js file are in the head section of HTML and not the body, like other js and css.

Reply:
Done.